test/unit_samples.jl

```diff
@@ -0,0 +1,31 @@
+# using Pkg
+# Pkg.activate("LazySets")
+# using Test
+
+using LazySets
+using Distributions
+using Polyhedra
+
+n = 5
+P1 = rand(BallInf, dim=n)
+A = [1.0  0.0;
+     0.0  1.0;
+    -1.0  0.0;
+     0.0 -1.0]
+b = [1.0, 2.0, 3.0, 4.0]
+P2 = HPolyhedron(A, b)
+P3 = HPolyhedron([0.0 0.1], [3.0])
+P4 = rand(Ball2, dim=n)
```

Comment:
```suggestion
P4 = Ball2{Float64}([0.2, -0.3, -1.1, 0.6, -0.7], 0.4)
```
I prefer to have deterministic tests where possible for reproducibility.
(Of course the sampling itself can be random.)